_gsocblogs/2024/blog_Key4hep_BraulioRivas.md

```diff
@@ -0,0 +1,131 @@
+| :-------------------------------------------------------------------- | :--------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Automated deploys via workflows                                       | [#16](https://github.com/key4hep/eede/pull/16) | I added 3 different workflow files: to deploy a "stable" version of eede. One to allow for pr previews and another to deploy a "release" version that has the latest features.                                                                                                          |
+| Redirect from key4hep.github.io/dmx to key4hep.github.io/dmx/main #19 | [#19](https://github.com/key4hep/eede/pull/19) | Because we have two versions: release and main, we can't have a single url to eede. So we have a general link https://key4hep.github.io/eede/ of eede that automatically redirects to the stable `main` version. But there is also a button that allows to go to the `release` version. |
+| Add testing for primitives, objects and tools                         | [#20](https://github.com/key4hep/eede/pull/20) | I added a basic test suite for some (now deprecated) graphing functions and for methods on objects.                                                                                                                                                                                     |
+| Fix previews workflow                                                 | [#22](https://github.com/key4hep/eede/pull/22) | I had some problems incorporating previews, so this PR tries to solve this issue (unsuccessfully).                                                                                                                                                                                      |
+| Configure previews with access tokens and check out to fork           | [#24](https://github.com/key4hep/eede/pull/24) | This PR finally fixed previews, by correctly configuring a GitHub environment that uses fined grained access tokens to checkout to the PR and deploy the code in the original repo.                                                                                                     |
+
+**Loading**
+
+After setting up the environment, I expanded the loading functionality. As explained above, at the beginning, eede could only
+load one type of datatype. However, there are many more, so I created a function called [`loadObjects`](https://github.com/key4hep/eede/blob/main/js/types/load.js#L47)
+that can load **all** of the datatypes from EDM4hep. But, internally is limited to load only those from the goals. Further
+work its easier because it only will be necessary to expand other modules, and not loading in order to load all the datatypes.
```

Comment:
You mention it in the table below, but I think this warrants another sentence here: This loading can also deal with different schema versions of edm4hep and has tooling to facilitate adding a new version. This is a very useful feature and you could highlight it explicitly in the text.